slam-libraries/viam-orb-slam3/setup_orbslam_macos.sh

```diff
@@ -0,0 +1,4 @@
+echo "Installing ORB_SLAM3 external dependencies"
+brew install cmake glew opencv eigen boost pangolin openssl
+brew link openssl --force
+sudo ln -s /opt/homebrew/Cellar/pangolin/0.8/lib/* /usr/local/lib/
```

Comment:
Without this, `orb_grpc_server` would compile but fail at runtime with a linking error, since the libraries for pangolin couldn't find each other. I'm not sure if this is a problem with how pangolin is built or with our brew package for it.